src/Uno.Toolkit.UI/Uno.Toolkit.UI.csproj

```diff
@@ -48,7 +48,6 @@
 
 	<ItemGroup>
 		<UpToDateCheckInput Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
-		<Content Include="Assets\**\*.png" />
```

Comment:
there is no need to remove this wildcard?
same thing for v2

Reply:
We don't need it anymore since the back icon pngs were the only assets being bundled